test-suite/templates/guess/js/original.expect

```diff
@@ -0,0 +1 @@
+nan
```

Comment:
Before we were expecting all of these but now just nan?

		"fs",
		"assert",
		"path",
		"nan",
		"buffer",
		"console",
		"dns",
		"child_process",

Reply:
I'm not comfortable removing the unit test. I feel like there is a bunch of parser functionality it is testing that I'm not confident is captured by the integration tests. Can we not remove it? Okay to not make it run for now, we can do that in a separate PR.

Reply:
restored and passing in [2c33a83](https://github.com/replit/upm/pull/154/commits/2c33a83c8ab221c17fec19ab6f16b5bc6e256970)